From d74bfbdb1797c15838286a7d105e406a43a2b3c4 Mon Sep 17 00:00:00 2001
From: Saravana Kannan <saravanak@google.com>
Date: Thu, 7 May 2020 11:59:43 -0700
Subject: [PATCH 307/830] ANDROID: GKI: driver core: Set fw_devlink=on by
 default
X-NVConfidentiality: public

Commit f4d0973e4fb0 ("ANDROID: of: property: Enable of_devlink by
default") enabled creating device links from DT info. However,
of_devlink command line argument has been replaced by fw_devlink.

So, set fw_devlink=on by default to get the same behavior as setting
of_devlink=on.

Bug: 143308885
Signed-off-by: Saravana Kannan <saravanak@google.com>
Change-Id: I5f63b0359082713bd7d3629abc14209f9667de3b
---
 drivers/base/core.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/base/core.c b/drivers/base/core.c
index 5d23cdb8885c..c5d981a1f551 100644
--- a/drivers/base/core.c
+++ b/drivers/base/core.c
@@ -1438,7 +1438,7 @@ static void device_links_purge(struct device *dev)
 	device_links_write_unlock();
 }
 
-static u32 fw_devlink_flags = DL_FLAG_SYNC_STATE_ONLY;
+static u32 fw_devlink_flags = DL_FLAG_AUTOPROBE_CONSUMER;
 static int __init fw_devlink_setup(char *arg)
 {
 	if (!arg)
-- 
2.17.1

